skipruntime-ts/ts/src/package.json

```diff
@@ -1,15 +1,9 @@
 {
   "type": "module",
   "scripts": {
-    "lint": "eslint ."
+    "lint": "eslint --config ../../../eslint.config.js ."
   },
   "devDependencies": {
-    "@eslint/js": "^9.9.1",
-    "@stylistic/eslint-plugin-js": "^2.6.4",
-    "@types/node": "^22.5.5",
-    "@typescript-eslint/parser": "^8.3.0",
-    "eslint-plugin-jsdoc": "^50.2.2",
-    "prettier": "3.2.5",
-    "typescript-eslint": "^8.3.0"
+    "@types/node": "^22.5.5"
```

Comment:
I don't want this comment to be blocking but I'm wondering whether we should really remove typing dependencies from this `package.json`. Won't they be needed by our users once the package is published?

My ChatGPT-helped understanding is that we should keep in `dependencies` packages that contain types that we expose. So let's keep things in `devDependencies` for now.

Reply:
Do you mean that perhaps `@types/node` should be in `dependencies` instead of `devDependencies`, or that it would be removed entirely? I tried to remove it entirely, but something run by `make check-all` adds it back.

Reply:
`@types/node` may be a special case, but I don't think what I wrote applies to any of our `devDependencies` yet. We don't re-expose types from our dependencies

Reply:
Ok, so to summarize: If we re-expose any types of a dependency, we should put it in `dependencies`, and if we only need them for building/linting/etc, we should put them in `devDependencies`.

Reply:
In the current state of the tip of this PR I have one remaining question about this: the examples/package.json has non-dev deps on sqlite3 and ws since it actually uses them at runtime, and also a dep on @types/sqlite3 which I'm unsure whether it should stay a non-dev dep of move to a devDep.